catalyst_voices_packages/catalyst_cardano_serialization/README.md

```diff
@@ -1 +1,179 @@
-# catalyst_cardano_serialization
+# Catalyst Cardano Serialization
+
```

Comment:
Let's add a `Table Of Content` and Requirements, Install, Support and License sections like here:
https://github.com/minikin/popover

Reply:
Updated